azure-pipelines.yml

```diff
@@ -9,27 +9,72 @@ variables:
   BUILD_NUMBER: $(Build.BuildNumber)
   BUILD_COMMIT: $(Build.SourceVersion)
 
+  # Build variables
+  mainBranchName: main                                                                  # Name of Git "main" branch
+  configuration: Release                                                                # Build configuration: 'Debug', 'Release'
+
+  # Reporting variables
+  areaPath: DevDiv\VS Client - Runtime SDKs\Android                                     # AzDo area path to log any issues
+
+  # Windows specific variables
+  windowsAgentPoolName: Maui-1ESPT                                                      # Windows VM pool name
+  windowsClassicInstaller: https://aka.ms/xamarin-android-commercial-d17-4-windows      # Windows Classic XA installer URL
+  
+  # macOS specific variables
+  macosAgentPoolName: Azure Pipelines                                                   # macOS VM pool name
+  macosImage: internal-macos12                                                          # macOS VM image name
+  macosClassicInstaller:  https://aka.ms/xamarin-android-commercial-d17-4-macos         # macOS Classic XA installer URL
+
 resources:
   repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
     - repository: internal-templates
       type: github
       name: xamarin/yaml-templates
       endpoint: xamarin
       ref: refs/heads/main
 
-jobs:
-  - template: build/ci/build.yml
-        
-  - ${{ if eq(variables['System.TeamProject'], 'devdiv') }}:
-    - template: sign-artifacts/jobs/v2.yml@internal-templates
-      parameters:
-        dependsOn: [ 'build' ]
-        condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
-
-    - template: compliance/sbom/job.v1.yml@internal-templates
-      parameters:
-        dependsOn: signing
-        condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
-        artifactNames: [ nuget-signed ]
-        packageName: androidx
-        packageFilter: '*.nupkg'
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:   
+    pool:
+      name: AzurePipelines-EO
+      image: 1ESPT-Windows2022
+      os: windows
```

Comment:
Since you are still passing pool information at the job you should be able to safely delete this top level `pool` value.

Reply:
Looks like the latest run failed, perhaps this was needed: 
> /v1/1ES.Pipeline.yml@1esPipelines (Line: 345, Col: 287): Unexpected parameter 'Please specify a windows pool for SDLSources stage. 

I ended up specifying this under the `sdl` element in other pipelines, though I'm not sure which approach is "more correct":
```yaml
sdl:
  sourceAnalysisPool:
    name: AzurePipelines-EO
    image: $(WindowsPoolImage1ESPT)
    os: windows
```